src/components/elements/ImageGallery/ImageGallery.tsx

```diff
@@ -3,17 +3,19 @@ import classNames from "classnames";
 import { DetailedHTMLProps, FC, HTMLAttributes, useEffect, useState } from "react";
 import { When } from "react-if";
 
-import Button from "@/components/elements/Button/Button";
 import FilterDropDown from "@/components/elements/TableFilters/Inputs/FilterDropDown";
 import Text from "@/components/elements/Text/Text";
-import { IconNames } from "@/components/extensive/Icon/Icon";
+import Icon, { IconNames } from "@/components/extensive/Icon/Icon";
 import Modal from "@/components/extensive/Modal/Modal";
 import { ModalId } from "@/components/extensive/Modal/ModalConst";
 import Pagination from "@/components/extensive/Pagination";
 import { VARIANT_PAGINATION_TEXT_16 } from "@/components/extensive/Pagination/PaginationVariant";
 import { useModalContext } from "@/context/modal.provider";
 import { Option } from "@/types/common";
 
+import Menu, { MenuItemProps } from "../Menu/Menu";
+import MenuColapse from "../Menu/MenuCollapse";
+import { MENU_PLACEMENT_BOTTOM_BOTTOM } from "../Menu/MenuVariant";
```

Comment:
imports should use non relative paths please

Reply:
done